webapp/app/forms/validators.py

```diff
@@ -115,3 +117,30 @@ def __call__(self, form, field):
         for char in input_str:
             if char not in VALID_UNLOCK_CODE_CHARACTERS:
                 raise ValidationError(_('validate.invalid-character'))
+
+class ValidDayOfBirth:        
+    def __call__(self, form, field):
+        raw_input_date = field._value()
+
+        if isinstance(raw_input_date, str):
+            day, month, year = raw_input_date.split('.')
+        elif isinstance(raw_input_date, list):
+            day, month, year = raw_input_date
```

Comment:
I think this will break if e.g. just the day is provided?

Reply:
(However: Thank for using that syntax, I've never seen that you can unpack a list like that. 👍 TIL^^)